.gitmodules

```diff
@@ -8,3 +8,6 @@
 [submodule "petsc"]
     path = petsc
     url = https://gitlab.com/petsc/petsc.git
+[submodule "moosetools"]
+	path = moosetools
+	url = git@github.com:idaholab/moosetools
```

Comment:
Oh, I take that back. This should be relative path. This makes it more versatile in that it'll work with HTTPS or SSH. Let's change that before merging!

Reply:
I'm not even sure I know what that should be, you'll have to play with it to make sure we get the right relative path. I'd try `url = ../moosetools` as a first attempt and see if it registers properly when you init.

Reply:
Didn't know this was a thing, it should be correct now. It is the same as large_media.